quesma/ingest/processor.go

```diff
@@ -834,11 +843,28 @@ func (ip *IngestProcessor) storeVirtualTable(table *chLib.Table) error {
 
 	table.Comment = "Virtual table. Version: " + now.Format(time.RFC3339)
 
+	var columnsToStore []string
+	for _, col := range table.Cols {
+		// We don't want to store attributes columns in the virtual table
+		if col.Name == chLib.AttributesValuesColumn || col.Name == chLib.AttributesMetadataColumn {
+			continue
+		}
+		columnsToStore = append(columnsToStore, col.Name)
+	}
+
+	// We always want to store timestamp in the virtual table
+	// if it's not already there
+	if slices.Index(columnsToStore, model.TimestampFieldName) == -1 {
```

Comment:
```suggestion
	if slices.Contains(columnsToStore, model.TimestampFieldName) {
```
Could we use this here instead?